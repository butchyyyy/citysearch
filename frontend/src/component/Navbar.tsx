import React from "react"
import { Navbar as BootstrapNavbar, NavbarBrand } from "reactstrap"

/** Application Navbar with application title */
const Navbar = () => (
    <BootstrapNavbar>
      <NavbarBrand color="light">City Search</NavbarBrand>
    </BootstrapNavbar>
)

export default Navbar
