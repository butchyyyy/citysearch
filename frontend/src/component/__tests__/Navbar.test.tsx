import Navbar from "component/Navbar"
import { shallow } from "enzyme"
import React from "react"

describe("Test Navbar Component", () => {
  const wrapper = shallow(<Navbar />)

  it("Test Navbar Snapshot", () => {
    expect(wrapper).toMatchSnapshot()
  })
})
