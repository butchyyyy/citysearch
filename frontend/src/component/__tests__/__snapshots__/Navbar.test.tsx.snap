// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Navbar Component Test Navbar Snapshot 1`] = `
<Navbar
  expand={false}
  tag="nav"
>
  <NavbarBrand
    color="light"
    tag="a"
  >
    City Search
  </NavbarBrand>
</Navbar>
`;
